mnist/web-ui/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN pip --no-cache-dir install \
         h5py \
         ipykernel \
         numpy \
-        tensorflow \
+        tensorflow==1.14.0 \
```

Comment:
@amygdala would list to pin the tensorflow to 1.7 in the PR https://github.com/kubeflow/examples/pull/658 since `from tensorflow.examples.tutorials.mnist import input_data` causing the problem, see more the #658 , could you please confirm if that works with 1.14.0 ? Thanks.